### ISSUE TITLE
Antoine Henry stepping down from the FR team

### ISSUE THREAD

**Issue by spapastamkou** [programminghistorian/jekyll#2767]
@icolab is stepping down for the FR team. Here is a list of what we have to do at the team level: 

- [x] Remove Google Groups access (@anisa-hawes)
- [x] Remove Google Analytics access (@acrymble) (Not applicable but I ping AC to be sure)
- [x] Remove Github Jekyll access @anisa-hawes 
- [x] Remove from ProgHist Ltd register of members (@drjwbaker)
- [x] Remove Twitter access (@anisa-hawes)
- [x] Change team: true to team: false, and add team_end: YYYY to the  _data/ph_authors.yml file to remove the person from the Project Team page (@anisa-hawes)
- [x] Open issues assigned to the editor: none (@spapastamkou)
- [x] Review service integrations page to ensure that they do not have any associated integrations tied to their accounts: @anisa-hawes (I'd say no but I ping AH to be sure)

**Comment by spapastamkou** [comment]
Merci et bonne chance dans tes futurs projets, @icolab !

**Comment by icolab** [comment]
Merci beaucoup @spapastamkou ! à toi aussi et bonne continuation à l'équipe PH !

**Comment by anisa-hawes** [comment]
Dear @spapastamkou 

I'm not able to remove @icolab from our organisation on GitHub – only the owners of our repository are able to do this. Perhaps @acrymble could help us? 

N.B. I am able to remove write access to outside collaborators, but I double checked and Antoine is not listed as having any access permissions on Jekyll or ph-submissions. I do not see @icolab listed as a member of /teams/french-team either.

Meanwhile, I've checked and can confirm that Antoine does not have access to our Twitter account, is not connected to any of our Service Integrations, and was not a member of our Google Group.

**Comment by acrymble** [comment]
@spapastamkou I don't think @icolab has jekyll access on this repository. It's not really necessary anymore for editors, so unless someone was added to it a long time ago, this isn't surprising.

**Comment by spapastamkou** [comment]
Thanks @anisa-hawes and @acrymble. First of all, I think we should update the [relevant wiki section](https://github.com/programminghistorian/jekyll/wiki/Admin-when-team-members-step-down)  (I can help to do this if we agree to the points to remove/change)
As for GitHub access: I am not sure what access we are talking about but I can still find icolab between the members I can assign this issue to (and I just did to test this!) so he does have rights to write and push to this repo as a team member. Whom should I ping about this? Zoe?

**Comment by spapastamkou** [comment]
@icolab J'ai fait un petit test en t'assignant ce ticket, pour voir si tu es toujours effectivement listé parmi les membres PH, tu n'as rien à faire en réalité:-)

**Comment by acrymble** [comment]
I think it’s because you have a French team somewhere. He should be able to leave that himself. @anisa-hawes can you update any onboarding guidelines to remove giving new members access to the Jekyll site. It isn’t needed anymore

**Comment by icolab** [comment]
Maybe it's because I do leave the organization directly from my Github account. Let me know if I need to do something.
Thank you and have a nice day

**Comment by anisa-hawes** [comment]
Thank you @acrymble and @spapastamkou,

I've updated the [Admin when team members step down](https://github.com/programminghistorian/jekyll/wiki/Admin-when-team-members-step-down) Wiki page, and will add this to my notes for onboarding too.

> * Remove Github Jekyll access (@acrymble) - only applicable to longer-serving members
> * Remove Github ph-submissions access (@anisa-hawes)

I remain slightly confused. Is it usually possible to assign someone outside the organisation to an Issue?
As I mentioned, I don't see @icolab listed as a member of /teams/french-team either.

The PR #2768 to update `ph_authors.yml` is ready for Review. I've tagged @marie-flesch, incase she'd like to take this opportunity to practice/see that process of updating the Project Team page in action again.

**Comment by spapastamkou** [comment]
@anisa-hawes 
Ok, I checked what Adam said and apparently the members can leave themselves the [project team](https://github.com/orgs/programminghistorian/teams/project-team/members). I think we should add this as an item in the list on the wiki, Anisa: precise it to the member who steps down and then verify it (you and/or the team leader). 

I believe Antoine appears still as assignee on this issue because I assigned him before he left. It is impossible to assign him another issue (I checked!) and it is also impossible to assign an issue to an external (I also checked with a C2DH colleague of mine for this issue). 
So we might be ok here; I think James updates the registry of members every 6 months or so, so perhaps we have no more to do in the current issue?
